bundles/k3d/dev/README.md

```diff
@@ -0,0 +1,11 @@
+# UDS Software Factory K3d Dev Bundle
+
+## Bundle Applications
+
+- [Minio](https://min.io/) - In-cluster S3 Object Storage (See below for more details)
+- [Postgres Operator](https://github.com/defenseunicorns/uds-package-postgres-operator) - In-cluster Postgresql Database
+- [UDS Core](https://github.com/defenseunicorns/uds-core) - Service Mesh, IdAM, Monitoring, Logging, MEtrics, UDS Policy Engine & Operator, Container Security, Backup and Restore
```

Comment:
```suggestion
- [UDS Core](https://github.com/defenseunicorns/uds-core) - Service Mesh, IdAM, Monitoring, Logging, Metrics, UDS Policy Engine & Operator, Container Security, Backup and Restore
```

Reply:
lol i fixed it in the one that actually has uds core. This is actually not true on the dev bundle and should be removed

Reply:
```suggestion
```